eos-add-flatpak-apps-repos.service

```diff
@@ -0,0 +1,20 @@
+# Add the stable Endless apps flatpak repositories after an OS upgrade
+
+[Unit]
+Description=Configure the stable Endless apps flatpak repositories
+DefaultDependencies=no
+Conflicts=shutdown.target
+Wants=local-fs.target
+After=local-fs.target
+
+# Only run on updates
+Before=multi-user.target systemd-update-done.service
+ConditionNeedsUpdate=/etc
```

Comment:
I think technically this should be `ConditionNeedsUpdate=/var` because that's where the repo config lives. (It does not matter much in practice because right now they're updated at the same time, but would be nice to get it right.)